codingTest_김준우/체육복 다른 풀이.js

```diff
@@ -0,0 +1,31 @@
+function solution(n, lost, reserve) {
+  let total = [];
+  let answer = 0;
+  for (let i = 0; i < n; i++) {
+    total[i] = 1;
+  }
```

Comment:
```suggestion
 const allStudent = Array(n).fill(1); 
```
`fill` 메소드를 사용하면 한 번에 배열의 값을 1로 조정할 수 있겠지요

Reply:
제가 풀었던 코드라 `total` 이 `allStudent`로 바뀌어 있네요 ㅜ 미안합니다